jvector-base/src/main/java/io/github/jbellis/jvector/util/AbstractLongHeap.java

```diff
@@ -84,32 +84,35 @@ protected long add(long element) {
     }
 
     /**
-     * Bulk-adds all elements from the given iterator to this heap, then re-heapifies
+     * Bulk-adds `elementsSize` elements from the given iterator to this heap, then re-heapifies
      * in O(n) time (Floyd's build-heap). For a proof explaining the linear time
      * complexity, see <a href="https://stackoverflow.com/a/18742428">this stackoverflow answer</a>.
```

Comment:
I think we should clarify that it bulk-adds the minimum between elementsSize and the number of elements in the iterator

Reply:
I like `pushMany`, and I'll modify this javadoc too.